pkg/apis/storage/validation/validation.go

```diff
@@ -403,7 +403,8 @@ func ValidateCSIDriver(csiDriver *storage.CSIDriver) field.ErrorList {
 
 // ValidateCSIDriverUpdate validates a CSIDriver.
 func ValidateCSIDriverUpdate(new, old *storage.CSIDriver) field.ErrorList {
-	allErrs := ValidateCSIDriver(new)
+	allErrs := field.ErrorList{}
+	allErrs = append(allErrs, apivalidation.ValidateCSIDriverName(new.Name, field.NewPath("name"))...)
 
```

Comment:
Do we need some validate test to ensure that when `CSIDriver` is update, it passes validation with potentially invalid value of `FSGroupPolicy`?

Reply:
I *think* I added this as https://github.com/kubernetes/kubernetes/pull/95739/commits/a8394a43a6ac3861cf86ab849d74fc15c06666bd .

Note that if we modify the spec to introduce this value afterward, the test will fail as the spec is immutable. Let me know if this is incorrect.